.eslintrc.json

```diff
@@ -3,10 +3,7 @@
         "browser": true,
         "es6": true
     },
-    "extends": [
-        "standard",
-        "plugin:react/recommended"
-    ],
+    "extends": ["plugin:prettier/recommended"],
```

Comment:
I think we should still extend a style guide. I like Airbnb but Standard is good too. The Prettier preset basically just enables Prettier to run as part of ESLint; it doesn't enforce any other standards on the code.

Reply:
I could just bring back standard then

Reply:
Turns out prettier does enforce with the eslint plugin, which is what I wanted. I got rid of my changes, but left prettier as the style config